src/features/pdf/PDFView.tsx

```diff
@@ -102,16 +78,17 @@ export const PDFView = ({ appName, appOwner }: PDFViewProps) => {
           {appOwner}
         </p>
       )}
-      {typeof pdfLayout !== 'undefined' ? (
-        <CustomPDFLayout layout={pdfLayout} />
-      ) : (
-        <AutomaticPDFLayout
-          pdfFormat={pdfFormat}
-          pageOrder={pageOrder}
-          hidden={hidden}
-          layouts={nodeLayouts}
-        />
-      )}
+      {topLevelComponents(pdfLayout).map((component) => (
+        <div
+          key={component.id}
+          className={css['component-container']}
+        >
+          <PDFComponent
+            component={component}
+            layout={pdfLayout}
+          />
+        </div>
+      ))}
```

Comment:
Wrapping the components is mostly to override the grid settings (which may be removed in the future). Also the `Form`-component has some other things like `ErrorReport` that I do not want to reuse. If i changed the page order to just the pdf-layout it would break the preview mode.